### PR TITLE
LPS-200168

### PR DESCRIPTION
**Pull request by MicaelleSilvaa**
Ticket: https://liferay.atlassian.net/browse/LPS-200168

### CODE REVIEW

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json [hunk 1]
```diff
@@ -311,6 +311,14 @@
 		"issueKey": "LPS-196611",
 		"to": "fetchByExternalReferenceCode(param#1#, param#0#)"
 	},
+	{
+		"classNames": [
+			"AssetCategoryLocalService",
+			"AssetCategoryLocalServiceUtil"
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/resources/dependencies/replacements.json:317]: @MicaelleSilvaa, shouldn't we also consider these changes in JSP files?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 2]
```diff
@@ -301,6 +301,14 @@ public class UpgradeCatchAllCheck {
 		_cpInstanceService.fetchByExternalReferenceCode(externalReferenceCode, companyId);
 	}
 
+	public void method(
+		long companyId, String externalReferenceCode, long groupId) {
+
+		_assetCategoryLocalService.fetchAssetCategoryByExternalReferenceCode(companyId, externalReferenceCode);
+		AssetCategoryLocalServiceUtil.fetchAssetCategoryByExternalReferenceCode(companyId, externalReferenceCode);
+		AssetCategoryLocalServiceUtil.fetchAssetCategoryByExternalReferenceCode(externalReferenceCode, groupId);
```
[the remark this answers is not in the document]
Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:309]: Resolved

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json [hunk 3]
```diff
@@ -311,6 +311,14 @@
...
+		],
+		"from": "fetchAssetCategoryByExternalReferenceCode(long, String)",
+		"issueKey": "LPS-196614"
```
[the remark this answers is not in the document]
Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/main/resources/dependencies/replacements.json:320]: Resolved

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava [hunk 4]
```diff
@@ -286,6 +286,14 @@ public class UpgradeCatchAllCheck {
 		_cpInstanceService.fetchByExternalReferenceCode(companyId, externalReferenceCode);
 	}
 
+	public void method(
+		long companyId, String externalReferenceCode, long groupId) {
+
+		_assetCategoryLocalService.fetchAssetCategoryByExternalReferenceCode(companyId, externalReferenceCode);
+		AssetCategoryLocalServiceUtil.fetchAssetCategoryByExternalReferenceCode(companyId, externalReferenceCode);
+		AssetCategoryLocalServiceUtil.fetchAssetCategoryByExternalReferenceCode(externalReferenceCode, groupId);
```
[the remark this answers is not in the document]
Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava:294]: This check is a case where the message is displayed

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json (hunk 1 above)
Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/main/resources/dependencies/replacements.json:317]: I've checked again and the changes don't apply to JSP

**Pull request closed without merging**